app/assets/stylesheets/articles.scss

```diff
@@ -953,123 +604,16 @@
-          1px solid $outline-color
-        );
-        @include themeable(
-          box-shadow,
-          theme-container-box-shadow,
-          $bold-shadow
-        );
-        &:hover {
-          @include themeable(
-            background,
-            theme-container-background-hover,
-            $light-gray
-          );
-        }
-        .sidebar-profile-snapshot-inner {
-          display: inline-block;
-          vertical-align: middle;
-          width: 100%;
-          line-height: 20px;
-          @include themeable(color, theme-container-color, $black);
-          &:hover {
-            opacity: 0.9;
-          }
-        }
-        .sidebar-profile-pic {
-          display: inline-block;
-          width: 50px;
-          height: 50px;
-          vertical-align: top;
-          overflow: hidden;
-          @include themeable(background, theme-container-background, $tan);
-          border-radius: 100%;
-          margin: 8px 5px 8px 12px;
-          img {
-            width: 100%;
-            border-radius: 100%;
-          }
-        }
-        .sidebar-profile-content {
-          display: inline-block;
-          width: calc(100% - 74px);
-          margin-top: 15px;
-          font-size: 13px;
-          vertical-align: middle;
-          text-align: left;
-          .sidebar-profile-name {
-            font-weight: 400;
-            font-size: 17px;
-          }
-          .sidebar-profile-username {
-            @include themeable(color, theme-secondary-color, $medium-gray);
-          }
-        }
-      }
-      margin-bottom: 20px;
+    
+    .sidebar-nav {
+      margin-bottom: 32px;
       .sidebar-nav-block {
-        padding: 0px;
-        width: calc(100% - 4px);
-        box-sizing: border-box;
-        &.principle-sidebar-nav-block {
-          padding: 9px 0px;
-          .sidebar-nav-element {
-            .sidebar-nav-link {
-              padding: 5px 8px;
-            }
-          }
-        }
         @media screen and (min-width: 950px) {
           .sidebar-nav-block-inner {
             height: 420px;
             overflow-y: scroll;
             overflow-x: hidden;
-          }
-        }
-        .sidebar-nav-header {
-          padding: 15px 10px;
-          font-size: 0.9em;
-          font-family: $monospace;
-          &.sidebar-nav-header-middle {
-            padding: 10px 10px;
-            text-decoration: underline;
-            @include themeable(color, theme-secondary-color, $medium-gray);
-          }
-          img {
-            width: 15px;
-            border: 0px;
-          }
-        }
-        .sidebar-nav-subheader {
-          padding: 10px;
-          font-weight: 500;
-          font-size: 0.9em;
-          @include themeable(
-            background,
-            theme-background,
-            darken($light-gray, 4%)
-          );
-          a {
-            @include themeable(color, theme-secondary-color, $black);
+            -webkit-overflow-scrolling: touch;
```

Comment:
We should really install [autoprefixer](https://www.npmjs.com/package/autoprefixer)

Reply:
I think we do have it, no? Or at least something that automatically adds vendor prefixes...

However, for this particular property, autoprefixer won't work. It's only webkit property and it doesn't really exist in official CSS spec. The reason why I added it is to implement smooth scrolling on iOS browsers. Everything that has `overflow: scroll` CSS property will not scroll smoothly on iOS browser. Honestly, it's not required but just a nice touch.